README.md

```diff
@@ -14,12 +14,33 @@ Welcome to NI's internal and external C# conventions and linter rules/plugins/to
 
 ## Quickstart
 
-Add a package reference to the package TODO_TBD.
+### Package Source
+Under the `packageSources` node, add a reference to the source of the package in your
+[`NuGet.Config`](https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior) file.
+
+```xml
+<packageSources>
+  <add key="<todo-tbd>" value="<todo-tbd>" />
+</packageSources>
+```
+
+### Package Reference
+Add a package reference to the package `NI.CSharp.Analyzers`.
 
 ```msbuild
-<PackageReference Include="" Version="" />
+<PackageReference Include="NI.CSharp.Analyzers" Version="0.1.0" />
```

Comment:
" Version="0.1.0" /> -- we should look to make something generic here that perhaps lists our versions of the package that are available. I don't think this is something we'll want to update every time we publish a new package.

Reply:
As of now, maybe `Version="<version-of-your-choice>"` or `Version="<latest-version-available>"`?

Or we could do `Version="1.x.x"`? This would mean on every major version update we would have to update this. Thoughts?

Reply:
I think <package-version> would be sufficient